.github/workflows/build.yml

```diff
@@ -1,6 +1,6 @@
 name: ci
 
-on:
+on: [push, pull_request]
   pull_request: {}
```

Comment:
You need to remove the following two lines

Reply:
lol ... I've just accepted your suggestions :( My bad on not checking if the other lines would be removed ;)

Reply:
But is interesting how tests are essentially not run in here.
@cortinico do you have ideas on why this is happening (or better not happening)?

Reply:
> @cortinico do you have ideas on why this is happening (or better not happening)?

I think it's either:

1. The settings for this repo are not allowing Github Actions coming from PRs. If this is the case we should probably ask @sjaensch to look into it.
2. This is the first PR that is adding a Github Action. It will work once you merge this.

Reply:
I'd vote for the first option ;)